packages/manager/modules/pci/src/projects/project/storages/databases/database/service-integration/add/add.html

```diff
@@ -65,6 +65,10 @@
             data-on-change="$ctrl.handleIntegrationChange(modelValue)"
             data-required
         >
+            <span data-ng-bind="$item.type"></span><br />
```

Comment:
```suggestion
            <span class="mb-2" data-ng-bind="$item.type"></span>
```

Reply:
If I add this class and delete the line breaker, I have not the result what I wished on two lines. Have you tested on your local machine ? I have all my information on one-line.